runtime/str.go

```diff
@@ -221,13 +221,13 @@ func strFind(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
 	s := toStrUnsafe(args[0]).Value()
 	l := len(s)
 	start, end := 0, l
-	if argc >= 3 {
+	if argc >= 3 && args[2] != None {
```

Comment:
Similar to adjustIndex, this block
```
        if i > seqLen {
		i = seqLen
	}
```
Fails for `"".find("", 1)` case, I think adjusting index for start and end can't be together.